File: src/test/java/net/dinkla/raytracer/samplers/HammersleyTest.java
package net.dinkla.raytracer.samplers;

import org.junit.jupiter.api.BeforeEach;

public class HammersleyTest extends GeneratorTest {

    @BeforeEach
    @Override
    public void initialize() {
        new Hammersley().generateSamples(NUM_SAMPLES, NUM_SETS , samples);
    }

}

File: src/test/kotlin/net/dinkla/raytracer/samplers/HammersleyTest.kt
package net.dinkla.raytracer.samplers

import org.junit.jupiter.api.BeforeEach

class HammersleyTest : AbstractGeneratorTest() {

    @BeforeEach
    override fun initialize() {
        Hammersley().generateSamples(NUM_SAMPLES, NUM_SETS, samples)
    }

}
